.env.example

```
@@ -0,0 +1,8 @@
+ORCH_API_KEY=secret
+PLAN_MAX_DIFF_LINES=20
+DEFAULT_TIMEZONE=America/New_York
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEFAULT_TIMEZONE key should go before the ORCH_API_KEY key

(UnorderedKey)